extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -235,11 +257,26 @@ export function createSection(context: SectionContext): azdata.GroupContainer {
 	});
 }
 
+function processRow(rowInfo: RowInfo, context: SectionContext): azdata.Component {
+	const rowItems: azdata.Component[] = [];
+	if ('fields' in rowInfo.fields[0]) { // rowInfo.fields is RowInfo[]
```

Comment:
which scenario are you trying to support here? fields inside a field? at the first glance, this doesn't seem correct for me.